packages/mongodb-rag-core/src/findContent/DefaultFindContent.ts

```diff
@@ -28,7 +28,7 @@ export const makeDefaultFindContent = ({
     let content = await store.findNearestNeighbors(embedding, {
       ...findNearestNeighborsOptions,
       filter: filters,
-      // TODO: need to add logic to pass limit to findNearestNeighbors
+      ...(limit ? { k: limit } : {}),
```

Comment:
can't u just do the following?

```suggestion
      k: limit,
```